CCP & CLKPSR defines are unknown within wiring.c

```
D:\Dev\Arduino\portable\packages\ATtiny10Core\hardware\avr\2.0.0\cores\tiny\wiring.c: In function 'init':
D:\Dev\Arduino\portable\packages\ATtiny10Core\hardware\avr\2.0.0\cores\tiny\wiring.c:8:3: error: 'CCP' undeclared (first use in this function)
   CCP = 0xD8;
   ^
D:\Dev\Arduino\portable\packages\ATtiny10Core\hardware\avr\2.0.0\cores\tiny\wiring.c:8:3: note: each undeclared identifier is reported only once for each function it appears in
D:\Dev\Arduino\portable\packages\ATtiny10Core\hardware\avr\2.0.0\cores\tiny\wiring.c:9:3: error: 'CLKPSR' undeclared (first use in this function)
   CLKPSR = 0b0000;
   ^
exit status 1
Fehler beim Kompilieren für das Board ATtiny10/9/5/4.

```

I think you missed to add the relevant #include in wiring.c

--- a/cores/tiny/wiring.c
+++ b/cores/tiny/wiring.c
@@ -2,6 +2,8 @@
   wiring.c
 */
 
+#include <Arduino.h>
+
 void init(void)
 {
   #if (F_CPU == 8000000L)

--- a/programmers.txt
+++ b/programmers.txt
@@ -4,3 +4,10 @@ usbasp.protocol=usbasp
 usbasp.program.protocol=usbasp
 usbasp.program.tool=avrdude
 usbasp.program.extra_params=-Pusb
+
+avrispmkii.name=AVRISP mkII
+avrispmkii.communication=usb
+avrispmkii.protocol=stk500v2
+avrispmkii.program.protocol=stk500v2
+avrispmkii.program.tool=avrdude
+avrispmkii.program.extra_params=-Pusb

